Pipfile

```diff
@@ -31,7 +31,7 @@ uvloop = "*"
 websockets = ">=9.1"
 httptools = "*"
 typing-extensions = "*"
-wikibaseintegrator = "*"
+wikibaseintegrator = ">=0.11.0"
```

Comment:
Wondering if we should have it be exactly v0.11.0 since it looks like there are likely going to be more changes in v0.12.0 https://github.com/LeMyst/WikibaseIntegrator#breaking-changes-in-v012
```suggestion
wikibaseintegrator = "==0.11.0"
```

Reply:
Yeah, I think pinning to 0.11 is wisest.

Reply:
(agree re: using sample files for basic etl validation)